client/order/index.js

```diff
@@ -135,221 +163,12 @@ jQuery( function ( $ ) {
 			return;
 		}
 
-		ReactDOM.render( <DisputeNotice chargeId={ chargeId } />, container );
+		ReactDOM.render(
+			<DisputedOrderNoticeHandler
+				chargeId={ chargeId }
+				onDisableOrderRefund={ disableWooOrderRefundButton }
+			/>,
```

Comment:
Curious to see if this name (and the props interface) works for y'all in context here.